ios/FluentUI/AvatarGroup/AvatarGroupModifiers.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+public extension AvatarGroup {
```

Comment:
why do we need this as separate extension?  it can't be part of avatar group directly?

Reply:
No technical reason, but to me it feels nice that there's a separate place where only view modifiers are listed. @rdeassis were there any additional reasons that you separated them out for `Avatar` or was it all code organization?